INSTALL.rst

```diff
@@ -77,7 +149,19 @@ article workflow.
 .. code-block:: console
     
```

Comment:
I would add here a comment about how to run it in docker too.

Reply:
xd, it's a few lines after :clown_face:

Reply:
:+1: